syntax = "proto3";

package grpc;

service ChatServer {
    rpc ChatStream (Empty) returns (stream Note);
    rpc SendNote (Note) returns (Empty);
}

message Empty {}

message Note {
    string name = 1;
    string message = 2;
}